beacon-chain/slasher/chunks.go

```diff
@@ -232,6 +233,43 @@ func (m *MinSpanChunksSlice) CheckSlashable(
 
 	surroundingVotesTotal.Inc()
 
+	// Both attestations should have the same type. If not, we convert both to Electra attestations.
+	unifyAttWrapperVersion(existingAttWrapper, incomingAttWrapper)
+
+	postElectra := existingAttWrapper.IndexedAttestation.Version() >= version.Electra
```

Comment:
`unifyAttWrapperVersion` converts `existingAttWrapper.IndexedAttestation` to `&ethpb.IndexedAttestationElectra` if it is not already the case.
So, how could `postElectra` not always be `true`?

Reply:
OK I get it, if they have both the `Phase0` type we don't convert them.